vlasiator.cpp

```diff
@@ -454,6 +454,7 @@ int main(int argn,char* args[]) {
    );
    
    const std::vector<CellID>& cells = getLocalCells();
+   computeCoupling(mpiGrid, cells, momentsGrid, &gridCoupling);
```

Comment:
This is now in conflict with how `balanceLoad()` calls the `computeCoupling()` with `technicalGrid` and `technicalGrid.coupling`